workerEngineReduce.py

```diff
@@ -1,131 +1,59 @@
 import logging
 
 
-if (config.CAREABOUTENGINE == 1):
-    engine = chess.engine.SimpleEngine.popen_uci(config.ENGINEPATH) #WHERE THE ENGINE IS ON YOUR COMPUTER
-    engine.configure({"Hash": config.ENGINEHASH})
-    engine.configure({"Threads": config.ENGINETHREADS})
-    logging.getLogger('chess.engine').setLevel(logging.INFO)
-
-#function to calculate percentage win rates for each colour, if more than 0 games
-def calc_percs(white, black, draws):
-
-    n = white + black + draws #wins + draws after move was played
-
-    if (n > 0) and (config.DRAWSAREHALF == 0):
-        total_games = n
-        white_perc = white / n
-        black_perc = black / n
-        draw_perc = draws / n
-        return white_perc, black_perc, draw_perc, total_games
-        
-    
-    else:
-        if (n > 0) and (config.DRAWSAREHALF == 1):
-                total_games = n
-                white_perc = (white + (0.5 * draws)) / n
-                black_perc = (black + (0.5 *draws)) / n
-                draw_perc = draws / n
-                return white_perc, black_perc, draw_perc, total_games
-
-        else:
-                return None, None, None, 0
-
-
-
-
-def calc_value(winRate, gamesPlayed, playRate, san, board): # p = white/black win rate and n = total games p was played, this function calculates values used in the potentcy score of each potential move. It do
-
-    if (gamesPlayed > config.MINGAMES) and (playRate >config.MINPLAYRATE): #total games move was played must be more than min games and min perc play rate (otherwise data is bad)
-        # print("check this", np.sqrt(winRate * (1-winRate) / gamesPlayed))
-        # print ("check this",st.norm.ppf(1 - config.ALPHA/2))
-        # print ("check this",winRate - st.norm.ppf(1 - config.ALPHA/2) * np.sqrt(winRate * (1-winRate) / gamesPlayed))
-        lb_value = max(0, winRate - st.norm.ppf(1 - config.ALPHA/2) * np.sqrt(winRate * (1-winRate) / gamesPlayed)) #lower bound wr at 95% confidence interval
-        ub_value = max(0, winRate + st.norm.ppf(1 - config.ALPHA/2) * np.sqrt(winRate * (1-winRate) / gamesPlayed)) #upper bound wr at
-    else:
-        winRate = 0
-        lb_value = 0
-        ub_value = 0
-
-
-
-    return winRate, lb_value, ub_value, gamesPlayed
-
-
-class WorkerPlay():
-    def __init__(self, fen, lastmove = '', san = '', ):
+class WorkerPlay:
+    def __init__(self, settings, status, engine, fen):
+        self.settings = settings
+        self.status = status
+        self.engine = engine
         self.fen = fen #fen is the game moves format needed to feed lichess api
         self.short_fen = fen[:-4]
-        self.lastmove = lastmove
-        self.san = san #san is the human written format for chess moves like 'Bc4'
-        self.terminal = '#' in san
         self.explored = False
         self.best_move = None
         
         self.board = chess.Board(fen)
         
         self.stats = self.call_api()
         self.parse_stats()
-           
-    def show(self):

```

Comment:
Those 2 methods were unused (and they were actually broken - there's no `display` method available anywhere, and the second method has a typo - `workerPlay` instead of `WorkerPlay`)